crates/router/src/connector/checkout/transformers.rs

```diff
@@ -95,7 +95,7 @@ impl<F, T>
 pub struct CardSource {
     #[serde(rename = "type")]
     pub source_type: CheckoutSourceTypes,
-    pub number: pii::Secret<String, pii::CardNumber>,
+    pub number: Option<cards::CardNumber>,
```

Comment:
Why was this updated to be an `Option`? Could you revert this to be a required value instead?

Reply:
not sure to be honest. there were so many files that I had to change that I lost overview a bit and this might have slipped in. Thanks for catching that.